signing_service/signing_service/signing_service.py

```diff
@@ -384,6 +419,34 @@ def _get_authentication_challenge_signature(self, authentication_challenge: byte
             authentication_challenge,
         )
 
+    @staticmethod
+    def _get_daily_transaction_threshold_file_path() -> PosixPath:
+        thresholds_directory = (Path.cwd()).joinpath('daily_tresholds')  # pylint: disable=no-member
+        thresholds_directory.mkdir(exist_ok=True)  # pylint: disable=no-member
+        daily_file = datetime.datetime.now().strftime('%Y-%m-%d')
+        thresholds_directory.joinpath(daily_file).touch(exist_ok=True)  # pylint: disable=no-member
+        thresholds_directory = thresholds_directory.joinpath(daily_file)
+        return thresholds_directory
+
+    def _get_signing_service_daily_transaction_sum_so_far(self) -> int:
+        threshold_file = self._get_daily_transaction_threshold_file_path()
+        try:
+            transaction_sum = int(threshold_file.read_text())  # pylint: disable=no-member
+        except ValueError:
+            threshold_file.write_text('0')  # pylint: disable=no-member
```

Comment:
don't you need to `open` / `close` the file?

Reply:
No, because we use `pathlib.Path` instead of `os.path`.